backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5622360_sys_gh1199_Add_Missing_Request_Windows .sql

```diff
@@ -0,0 +1,240 @@
+-- 2022-01-20T15:44:05.679Z
```

Comment:
@riadmen why do we need this script? what's done here?

i thought the only thing we need is to `UPDATE AD_Menu SET IsCreateNew='Y'.....`....